samples/bluetooth/common/address_verification.c

```diff
@@ -0,0 +1,51 @@
+/* Copyright (C) 2025 Alif Semiconductor - All Rights Reserved.
+ * Use, distribution and modification of this code is permitted under the
+ * terms stated in the Alif Semiconductor Software License Agreement
+ *
+ * You should have received a copy of the Alif Semiconductor Software
+ * License Agreement with this file. If not, please write to:
+ * contact@alifsemi.com, or visit: https://alifsemi.com/license
+ */
+
+
+#include <zephyr/kernel.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/random/random.h>
+#include "address_verification.h"
+#include "gapm_le.h"
+
+LOG_MODULE_REGISTER(address, LOG_LEVEL_DBG);
+
+gapm_config_t *local_gapm_cfg;
+
+uint8_t address_verif(uint8_t SAMPLE_ADDR_TYPE, gapm_config_t *gapm_cfg)
```

Comment:
```suggestion
uint8_t address_verif(uint8_t sample_addr_type, gapm_config_t *gapm_cfg)
```
Parameters are usually lower letter.

Reply:
It is a constant and defined with capital letters.

Reply:
nevermind